source/helpers/moduleResolver.ts

```diff
@@ -73,7 +73,7 @@ export function readSystemAttributes(): {[index: string]: any} {
     'gc.heap.used': mem.heapUsed,
     'node.env': process.env.NODE_ENV,
     'debug.port': process.debugPort,
-    'backtrace.version': VERSION,
+    'application.version': VERSION,
```

Comment:
application.version is reserved for current application version, not for backtrace-node version. `backtrace.version` is a dedicated attribute for this purpose.

Reply:
I'm a bit confused here. In the [RFC](https://docs.google.com/document/d/1_fCK4NHFiCjb4JmaO_R5drX8L4_OQhSneIuBdQ2z7Hs/) the `application.version` is a required attribute and states `the application version specified by the developer`,
What should it be?

Reply:
`application.version` will now show the backtrace-node version, not app version. If I gonna release an express web API in version 3.2.0, instead of adding application.version 3.2.0 (which is expected behavior) we gonna send a backtrace-node version which is 1.2.0 now

Reply:
It appears application.version _is_ backtrace.version. I found here that you've passed application.version as the backtrace package.version (ex. 1.2.0)
<img width="976" alt="Screen Shot 2022-01-26 at 12 07 58 PM" src="https://user-images.githubusercontent.com/47638521/151238977-3c487d98-a8de-4164-9e1d-b49a0fbcdcb3.png">

Reply:
Ok, so to confirm, current change is expected behavior. `VERSION` is the backtrace version, being passed for the `application.version` field

![image](https://user-images.githubusercontent.com/47638521/151239331-4cde15f3-0419-446e-888e-fb233bfe7da6.png)

Reply:
Update: I had things backwards - pushed changes.